spec/index.html

```diff
@@ -1152,6 +1152,23 @@ <h3>The XML Schema Built-in Datatypes</h3>
         model.</li>
     </ul>
 
+    <p class="note">As mentioned in the table above, the <a>value spaces</a>
+      of <a data-cite="xmlschema11-2#double"><code>xsd:double</code></a> and
+      <a data-cite="xmlschema11-2#float"><code>xsd:float</code></a> do not
+      include all decimal numbers. The reason is that, for every literal of
+      any of these two datatypes, the value given as the <a>lexical form</a>
+      of the literal is rounded to a value that can be represented as an
```

Comment:
```suggestion
      include all decimal numbers. The <a>lexical form</a> is mapped to a value that
      is rounded to a value represented by
```

Reply:
Shortening the text as you suggest may introduce confusion. Readers may ask themselves: ' "The lexical form" of what? '

Reply:
Then how about just removing "The reason is that, "

Reply:
> Then how about just removing "The reason is that, "

Done.